PWGCF/MultiparticleCorrelations/Core/MuPa-DataMembers.h

```diff
@@ -12,11 +12,13 @@
 #ifndef PWGCF_MULTIPARTICLECORRELATIONS_CORE_MUPA_DATAMEMBERS_H_
 #define PWGCF_MULTIPARTICLECORRELATIONS_CORE_MUPA_DATAMEMBERS_H_
 
-// Remarks:
-// 0. Starting with C++11, it's possible to initialize data members at
-// declaration, so I do it here
-// 1. Use //!<! for introducing a Doxygen comment interpreted as transient in
-// both ROOT 5 and ROOT 6.
+// General remarks:
+// 0. Starting with C++11, it's possible to initialize data members at declaration, so I do it here
+// 1. Use //!<! for introducing a Doxygen comment interpreted as transient in both ROOT 5 and ROOT 6.
+// 2. If I hit at the compilation error: Framework/StructToTuple.h:286:6: error: only 99 names provided for structured binding
+//    2a) As far as I can tell, that means that sum of individual data members + struct fields + individual configurables > 99
+//    2b) Therefore, wrap up all data members in some struct fields + use in instead of individual configurables arrays whenever possible.
```

Comment:
Consider the possibility of using `ConfigurableGroup`

Reply:
Thanks, I was not aware of that possibility.